onnxruntime/core/graph/contrib_ops/shape_inference_functions.cc

```diff
@@ -3,12 +3,17 @@
 
 #include "core/graph/contrib_ops/shape_inference_functions.h"
 #include <onnx/defs/shape_inference.h>
+#include <iostream>
 
 namespace onnxruntime {
 namespace contrib {
 void EmbedLayerNormalizationShapeInference(::ONNX_NAMESPACE::InferenceContext& ctx) {
   propagateElemTypeFromInputToOutput(ctx, 2, 0);
-  if (!ctx.getOutputType(1)) {
+  auto mask_index_type = getAttribute(ctx, "mask_index_type", 1);
+  // std::cout << "Num outputs: " << (ctx.getNumOutputs()) << std::endl;
```

Comment:
Remove commented code.

Reply:
I'll remove it once I finish resolving the Windows GPU CI Pipeline error